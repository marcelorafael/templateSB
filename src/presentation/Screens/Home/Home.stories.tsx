import { withKnobs } from '@storybook/addon-knobs'
import { storiesOf } from '@storybook/react-native'
import React from 'react'
import { Home } from './Home'

storiesOf('Screens/Home', module)
  .addDecorator(withKnobs)
  .add('default', () => <Home />)
