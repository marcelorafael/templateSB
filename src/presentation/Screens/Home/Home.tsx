import React from 'react'
import { Container, Title } from './Home.styles'

const Home: React.FC = () => {
  return (
    <Container>
      <Title>Home</Title>
    </Container>
  )
}
const HomeMemo = React.memo(Home)
export { HomeMemo as Home }
