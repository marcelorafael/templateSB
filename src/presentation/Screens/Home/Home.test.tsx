import React from 'react'
// import { renderWithProvider } from '../../tests/utils'
import { render } from '@testing-library/react-native'
// import renderer from 'react-test-renderer'
import { Home } from './Home'
import 'setimmediate'

describe('Screens/Home', () => {
  it('should match snapshot', () => {
    const sut = <Home />

    const renderScreen = render(sut)

    expect(renderScreen).toMatchSnapshot()
  })
})
