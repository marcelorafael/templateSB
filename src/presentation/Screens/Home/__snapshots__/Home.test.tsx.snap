// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens/Home should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#030009",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#FFF",
        },
      ]
    }
  >
    Home
  </Text>
</View>
`
